README.md

```diff
@@ -130,32 +138,38 @@ The following snippet demonstrates how to create a small data curation pipeline
 dataset = download_common_crawl("/datasets/common_crawl/", "2021-04", "2021-10", url_limit=10)
 # Build your pipeline
 curation_pipeline = Sequential([
+  # Fix unicode
   Modify(UnicodeReformatter()),
+  # Discard short records
   ScoreFilter(WordCountFilter(min_words=80)),
+  # Discard low-quality records
   ScoreFilter(FastTextQualityFilter(model_path="model.bin")),
+  # Discard records from irrelevant tasks
   TaskDecontamination([Winogrande(), Squad(), TriviaQA()])
 ])
-# Curate your dataset
+# Execute the pipeline on your dataset
 curated_dataset = curation_pipeline(dataset)
 ```
 
-## Explore NeMo Curator Tutorials
+### Explore NeMo Curator Tutorials
 
-To get started with NeMo Curator, you can follow the tutorials available here: [Tutorials]
-(https://github.com/NVIDIA/NeMo-Curator/tree/main/tutorials). These tutorials include:
+To get started with NeMo Curator, you can follow the tutorials [available here](https://github.com/NVIDIA/NeMo-Curator/tree/main/tutorials). These tutorials include:
 
-- `tinystories` which focuses on data curation for training from scratch.
-- `peft-curation` which focuses on data curation for parameter-efficient fine-tuning use-cases.
+- [`tinystories`](https://github.com/NVIDIA/NeMo-Curator/tree/main/tutorials/tinystories) which focuses on data curation for training LLMs from scratch.
+- [`peft-curation`](https://github.com/NVIDIA/NeMo-Curator/tree/main/tutorials/peft-curation) which focuses on data curation for LLM parameter-efficient fine-tuning (PEFT) use-cases.
+- [`distributed_data_classification`](https://github.com/NVIDIA/NeMo-Curator/tree/main/tutorials/distributed_data_classification) which focuses on using the quality and domain classifiers to help with data annotation and blending.
```

Comment:
Nit: There is no data blending that occurs in this tutorial.
```suggestion
- [`distributed_data_classification`](https://github.com/NVIDIA/NeMo-Curator/tree/main/tutorials/distributed_data_classification) which focuses on using the quality and domain classifiers to help with data annotation.
```

Reply:
Correct, the tutorial's overview says it's mean to "help" with data annotation and blending, which is why I thought to include that here.

I changed the wording so it only mentions annotation.